resources/assets/js/components/JobBuilder/JobBuilderStep.tsx

```diff
@@ -131,10 +131,10 @@ const JobBuilderStep: React.FunctionComponent<JobBuilderStepProps> = ({
   useEffect((): void => {
     if (jobId !== null && job === null) {
       nprogress.start();
-    } else {
+    } else if (currentPage !== "intro") {
```

Comment:
Whats the intention of this line?

Reply:
So if your past the Intro page in the JPB and decide to go back (for whatever reason), the progress bar was finishing when the job finished loading, but not when the manager profile finished loading. Therefore there is a `nprogress.done()` that finishes after the manager's done loading in the `JobBuilderIntro.tsx` page.